keyboards/dekunukem/duckypad/keymaps/m4cs/config.h

```diff
@@ -0,0 +1,62 @@
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif // !NO_DEBUG
+#if !defined(NO_PRINT) && !defined(CONSOLE_ENABLE)
+#define NO_PRINT
+#endif // !NO_PRINT
+
+#define NO_ACTION_FUNCTION
```

Comment:
```suggestion
#pragma once
```

The rest shouldn't be needed if console isn't enabled. 
And needs a license header.